torch/autograd/__init__.py

```diff
@@ -265,6 +266,10 @@ def grad(
             cliffs. Please use ``torch._C._debug_only_display_vmap_fallback_warnings(True)``
             to show any performance warnings and file an issue on github if warnings exist
             for your use case. Defaults to ``False``.
+        zero_grad_unused (bool, optional): If ``True``, the gradient for inputs that were
+            not used when computing outputs will be set to zero instead of None. This can
+            be helpful in scenarios that involve multiple differentiations. Defaults to ``False``.
```

Comment:
I'd update the last sentence to: `This can be useful when computing higher-order derivatives`

Reply:
The comments updated as suggested. Please see https://github.com/pytorch/pytorch/pull/97015/commits/36b141ad1709c895899346e33ad7e6d54d128989